appraisal.lic

```diff
@@ -96,11 +96,12 @@ class Appraisal
   end
 
   def appraise_item?(item)
-    @equipment_manager.get_item?(item)
-    bput("appraise #{item.short_name} quick", 'Roundtime')
-    waitrt?
-    pause 1
-    @equipment_manager.empty_hands
+    if @equipment_manager.get_item?(item)
+      bput("appraise #{item.short_name} quick", 'Roundtime')
+      waitrt?
+      pause 1
+      @equipment_manager.empty_hands
+    end
```

Comment:
I wouldn't mind seeing an else that reports the problem to the user, but it's not a big deal either.

Reply:
Agree, thankfully the `equipment_manager.get_item?` method already logs when it doesn't get the item: https://github.com/rpherbig/dr-scripts/blob/master/equipmanager.lic#L230

The appraise script just needed to be smart enough to skip trying to appraise the non-existant item.

Thanks!

Reply:
Good point!